### DISCUSSION TITLE
有计划出VSCode版本吗？

### DISCUSSION THREAD

**Discussion by nicholaschen-cn** [Keldos-Li/typora-latex-theme#68] [Q&A] [answered]
您好，请问您有计划将主题迁移到VSCode吗？
现在Typora已经开始收费了，使用不方便。

**Comment by RalXYZ** [comment, marked as answer]
我们目前有这样的考虑，但还没研究清楚具体方案。  
原因是，根据我们的调查，我们发现 VS code 没有支持自定义 CSS 的官方解决方案。看起来不少人都是通过装插件来解决的 markdown 浏览。所以，就产生了以下两个问题：
1. 一些插件不能加载自定义 CSS
2. 即使支持加载，由于有很多预览插件，它们对自定义 CSS 的规则可能有一些差别；但既然我们要支持 vs code ，就要尽可能做到通用性

当然，以上两个问题也不是不可解。**只要我们确认一个有开发者在稳定维护、使用较为广泛，且能自定义 CSS 的预览插件，就可以进行迁移了。**  

以上的调查发生在几个月前，当前的情况可能已经与当时有区别；且我们的调查也可能存在偏差。
对此，您有什么建议呢？比如您可以提出一个支持 VS code 的方案，然后我们再进行接下来的讨论。

**Reply by GiggleDing** [reply]
您好，我在 VSCode 官网找到了[在 markdown 中使用自定义 CSS](https://code.visualstudio.com/docs/languages/markdown#_using-your-own-css) 的相关内容。
